### DISCUSSION TITLE
[Matter] All endpoints are exposed with name generic "Relay1" name

### DISCUSSION THREAD

**Discussion by lboue** [arendst/Tasmota#18714] [Support request]
### PROBLEM DESCRIPTION

1. All endpoints have the same name (generic "Relay1") in via the Matter protocol.
2. Some buttons are grayed out.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): Atom Lite
- [x] Tasmota binary firmware version number used: 12.5.0.3 by Theo Arends
  - [x] Pre-compiled
  - [ ] Self-compiled
- [x] Flashing tools used: [[_____](https://tasmota.github.io/install/)](https://tasmota.github.io/install/)
- [x] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
22:11:06.391 CMD: Backlog Template
22:11:06.422 MQT: stat/tasmota_D686D0/RESULT = {"NAME":"M5Stack Atom Lite","GPIO":[1,1,1,1,1,1,1,1,1056,1,1,1,1,1,1,1,0,1,1,1,0,1,640,1376,0,0,0,0,608,1,1,1,1,0,0,32],"FLAG":0,"BASE":1}
22:11:19.169 CMD: Module
22:11:19.176 MQT: stat/tasmota_D686D0/RESULT = {"Module":{"0":"M5Stack Atom Lite"}}
22:11:24.158 CMD: GPIO 255
22:11:24.168 MQT: stat/tasmota_D686D0/RESULT = {"GPIO0":{"0":"None"},"GPIO1":{"0":"None"},"GPIO2":{"0":"None"},"GPIO3":{"0":"None"},"GPIO4":{"0":"None"},"GPIO5":{"0":"None"},"GPIO6":{"0":"None"},"GPIO7":{"0":"None"},"GPIO8":{"0":"None"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO11":{"0":"None"},"GPIO12":{"1056":"IRsend1"},"GPIO13":{"0":"None"},"GPIO14":{"0":"None"},"GPIO15":{"0":"None"},"GPIO16":{"0":"None"},"GPIO17":{"0":"None"},"GPIO18":{"0":"None"},"GPIO19":{"224":"Relay1"},"GPIO20":{"0":"None"},"GPIO21":{"225":"Relay2"},"GPIO22":{"0":"None"},"GPIO23":{"0":"None"},"GPIO24":{"0":"None"},"GPIO25":{"0":"None"},"GPIO26":{"640":"I2C SDA1"},"GPIO27":{"1376":"WS28121"},"GPIO32":{"608":"I2C SCL1"},"GPIO33":{"0":"None"},"GPIO34":{"0":"None"},"GPIO35":{"0":"None"},"GPIO36":{"0":"None"},"GPIO37":{"0":"None"},"GPIO38":{"0":"None"},"GPIO39":{"32":"Button1"}}
```

### TO REPRODUCE
_Steps to reproduce the behavior:_
-  Setup endpoint names
-  Setup Matter with one bridged device ([Nous A5T WiFi Smart Power Strip with Tasmota](https://nous.technology/product/a5t.html)) with 4 relays. Setup relay proxy to `0,192.168.1.103`

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._
Expose endpoint name instead of generic "Relay1".

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._
![image](https://github.com/arendst/Tasmota/assets/938089/ad279d09-6009-474d-a62a-99903be07938)

![image](https://github.com/arendst/Tasmota/assets/938089/cd71b05b-9b96-4105-b9f9-1e92dd98a1ac)
![image](https://github.com/arendst/Tasmota/assets/938089/1e275cfb-0445-404b-b295-b70808b2efda)

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

[matter-e87f29d9b987f939a7c2ff854db369c3-Relay1-7077575bbd656af69587555ed36b3be3.json.txt](https://github.com/arendst/Tasmota/files/11537100/matter-e87f29d9b987f939a7c2ff854db369c3-Relay1-7077575bbd656af69587555ed36b3be3.json.txt)

Matter fabric config:
```
        "0/29/65532": 0,
        "0/29/65533": 1,
        "0/40/0": 1,
        "0/40/1": "Tasmota",
        "0/40/2": 65521,
        "0/40/3": "Atom Lite",
        "0/40/4": 32768,
        "0/40/5": "Relay1",
        "0/40/6": "**REDACTED**",
        "0/40/7": 0,
        "0/40/8": "ESP32-PICO-D4 rev.1",
        "0/40/9": 1,
        "0/40/10": "12.5.0.3",
        "0/40/15": "4C:75:25:D6:86:D0",
        "0/40/18": "4C:75:25:D6:86:D0",
        "0/40/19": {
          "caseSessionsPerFabric": 3,
          "subscriptionsPerFabric": 3
        },
```

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by s-hadinger** [comment]
Thanks. Relay names are not yet implemented. iOS did not really care about the name. I will fix that later

**Reply by lboue** [reply]
Thanks.

**Reply by lboue** [reply]
Since Tasmota dev has switch to Matter bridge mode, the display of names is technically possible.
This is the case with Hue Bridge for example.

More info here: https://github.com/arendst/Tasmota/discussions/18873

**Comment by s-hadinger** [comment]
Fixed with https://github.com/arendst/Tasmota/pull/18897
